assets/js/test-unit/theme/common/state-country.spec.js

```diff
@@ -1,18 +1,16 @@
 import $ from 'jquery';
 import '../../../theme/global/jquery-migrate';
-//import { api } from '@bigcommerce/stencil-utils';
+
 jest.mock('@bigcommerce/stencil-utils');
 import utils from '@bigcommerce/stencil-utils';
+
 const api = utils.api;
-import modalFactory, { alertModal } from '../../../theme/global/modal';
-import foundation from '../../../theme/global/foundation';
+import { alertModal } from '../../../theme/global/modal';
 import stateCountry from '../../../theme/common/state-country';
 
-//console.log(utils);
-//jest.mock('api.country');
-
 describe('StateCountry', () => {
-    let $countryElement, $stateElement;
+    let $countryElement; let
```

Comment:
just a remark: in my opinion, the second `let` would be read better from the next line

Reply:
that's true, i missed this